unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -905,11 +905,9 @@ notifyUser dir o = case o of
           ( P.syntaxToColor . prettyHashQualified' . fmap Name.fromSegment $ hq,
             isBuiltin r
           )
-        ShallowBranchEntry ns _ count ->
+        ShallowBranchEntry ns _ ->
           ( (P.syntaxToColor . prettyName . Name.fromSegment) ns <> "/",
-            case count of
-              1 -> P.lit "(1 definition)"
-              _n -> P.lit "(" <> P.shown count <> P.lit " definitions)"
+            "(namespace)"
```

Comment:
TODO: double-check how this affects `ls` output, and what the best way to maintain this behaviour is.